server/embed/config_test.go

```diff
@@ -152,6 +157,7 @@ func TestConfigFileFeatureGates(t *testing.T) {
 			expectedFeatures: map[featuregate.Feature]bool{
 				features.StopGRPCServiceOnDefrag: false,
 				features.DistributedTracing:      false,
+				features.InitialCorruptCheck:     false,
```

Comment:
Can we add a test for the enablement of the flag?

Reply:
Added